app/src/test/java/backgammon/DoublingCubeTests.java

```diff
@@ -0,0 +1,16 @@
+package backgammon;
+
+public class DoublingCubeTests {
+  public static void main(String[] args){
+
+    DoublingCube jim = new DoublingCube();
+
+    System.out.println(jim.getCubeValue());
+
+    jim.doubleCubeValue();
+    jim.doubleCubeValue();
+
+    System.out.println(jim.getCubeValue());
```

Comment:
Close but not even quite.
Define a method with a descriptive name such as `canDoubleDoublingCube` with the necessary `@Test` annotation. then use the `assertEquals()` method to test the value of the doubling cube.

You pass in your expected value, then your actual result from the cube, and optionally a string
that displays what the test is supposed to accomplish.